elastic/elasticsearch.go

```diff
@@ -3,23 +3,25 @@ package elasticsearch
 import (
 	"context"
 	"fmt"
-	"log"
 	"net/url"
-	"os"
 
 	"github.com/aws/aws-sdk-go/aws/credentials"
 	"github.com/olivere/elastic"
 	aws "github.com/olivere/elastic/aws/v4"
+	"github.com/tozny/utils-go/logging"
 )
 
 // ElasticClient wraps a Client for Elasticsearch interactions. The struct also includes a logger, that will get configured, in instantiation methods.
 type ElasticClient struct {
 	*elastic.Client
-	*log.Logger
+	*logging.ServiceLogger
```

Comment:
Now that this is available - suggest changing to use the `logging.Logger` interface instead.